web/src/components/netflow-table/netflow-table.tsx

```diff
@@ -103,16 +103,26 @@ const NetflowTable: React.FC<{
     bodyContent = getSortedFlows().map(f => <NetflowTableRow key={f.key} flow={f} columns={columns} size={size} />);
   }
 
+  const width = columns.reduce((prev, cur) => prev + cur.width, 0);
+  if (width === 0) {
+    return null;
+  }
+
   return (
-    <TableComposable aria-label="Misc table" variant="compact">
-      <NetflowTableHeader
-        onSort={onSort}
-        sortDirection={activeSortDirection}
-        sortIndex={activeSortIndex}
-        columns={columns}
-      />
-      <Tbody>{bodyContent}</Tbody>
-    </TableComposable>
+    <OuterScrollContainer>
+      <InnerScrollContainer>
+        <TableComposable aria-label="Netflow table" variant="compact" style={{ minWidth: `${width}em` }} isStickyHeader>
```

Comment:
My last issues is on this `minWidth` when I get error / loading / empty states :
![image](https://user-images.githubusercontent.com/91894519/152188127-8226f9b8-5f3c-406a-8b1f-2360a8d569da.png)

We should either remove all table scrollbars in these cases or entirely remove the table (seems to be the way it's done on pods side)

Reply:
it should be fixed via  b327079